src/toplevel.pl

```diff
@@ -170,8 +170,8 @@
 %% Enable op declarations with lists of operands, i.e.,
 %% :- op(900, fy, [$,@]).
 
-user:term_expansion((:- op(Pred, Spec, Ops)), OpResults) :- 
```

Comment:
That `Pred` should be `Pri` for priority.

Reply:
Well, that was there before this PR, but I guess I could fix it too.